wallet/src/Pos/Wallet/Web/Pending/Worker.hs

```diff
@@ -167,6 +167,8 @@ startPendingTxsResubmitter
     => (TxAux -> m Bool)
     -> m ()
 startPendingTxsResubmitter submitTx =
+    -- TODO: [CSL-198] Catch and report non-critical errors, as
+    -- 'onNewSlot' no longer swallows all exception and rethrows them instead.
```

Comment:
It seems that this worker already catches exceptions which may occur during tx submission (see code in `submitAndSavePtx`), so I think nothing should be changed.